app/table/columns/common.tsx

```diff
@@ -12,15 +12,19 @@ import { filesize } from 'filesize'
 import { Truncate } from '~/ui/lib/Truncate'
 
 import { EmptyCell } from '../cells/EmptyCell'
-import { TwoLineCell } from '../cells/TwoLineCell'
 
 // the full type of the info arg is CellContext<Row, Item> from RT, but in these
 // cells we only care about the return value of getValue
 type Info<T> = { getValue: () => T }
 
 function dateCell(info: Info<Date>) {
   const date = info.getValue()
-  return <TwoLineCell value={[format(date, 'MMM d, yyyy'), format(date, 'p')]} />
+  return (
+    <time dateTime={date.toISOString()} className="flex flex-wrap gap-x-2">
+      <span>{format(date, 'PP')}</span>
+      <span className="text-quaternary">{format(date, 'p')}</span>
+    </time>
+  )
```

Comment:
I'm realizing `date-fns` is very literal and doesn't seem to do any locale-based formatting. I think we should use `toLocaleString()` and get decent results, i.e., the same as we're currently getting in English but better for other languages. Something like this:

```ts
const dateStr = date.toLocaleString(undefined, { month: 'short', day: 'numeric', year: 'numeric' })
const timeStr = date.toLocaleString(undefined, { hour: 'numeric', minute: 'numeric' }))
```

<img width="573" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/8ac4d0ff-b96d-4f88-a993-a60fdface1a9">

In the math function we use `navigator.language` where I've put `undefined` here, but it seems like `undefined` is really the right way. The only problem is that the way I've tested this is to mock `navigator.language`, so if we use `undefined` instead that approach won't work in the tests. Maybe we should be using playwright instead to test the locale rendering. They make it really easy:

```ts
import { test, expect } from '@playwright/test';

test.describe('french language block', () => {

  test.use({ locale: 'fr-FR' });

  test('example', async ({ page }) => {
    // ...
  });
});
```

https://playwright.dev/docs/test-use-options#configuration-scopes

To summarize, I think you should:

1. Replace these `date-fns` `format` calls with `toLocaleString(undefined, ...)`
2. Check a displayed date in exactly one table in `de-DE` or whatever using the Playwright local setting feature
3. Don't worry about math, I'll make an issue to switch those to use `undefined` instead of `navigator.language` and change the way we test it